### ISSUE TITLE
Add option to disable translated names on the map

### ISSUE THREAD

**Issue by mvglasow** [organicmaps/organicmaps#4043]
By default, Organic Maps will try to display feature names in the UI language. For some features (such as cities), this is the only name displayed and the local name is used only as a fallback if no translation exists. For other features (such as roads), the translated name is displayed alongside the local name.

Different people will likely have different preferences here. For me, I would prefer local names over translated ones, so I can easily match them with names on signs (which will mostly be in the local language). I find it confusing to have just the translated name but not the local one – some pairings can be hard to match up if you’re not familiar with the geography, history and/or language of the place, e.g. Rijeka vs. Fiume. (For more such examples, set the UI language to German and browse some of the territories ceded by Germany after WWI or WWII – not getting lost there may become a real challenge). Showing both names takes up screen real estate (and the translation may push the local name off-screen).

For that reason, I would like to be able to choose between local and translated names.

Note that this is not the same as transliterating names (which already can enabled/disabled via a preference). I would like to see local names if they use Latin script, but a Latin transliteration of other scripts (as I don’t read, say, Georgian or Hebrew or Thai). Maybe it would make sense to combine these two into a preference with three options:

* always use local names
* transliterate names from non-Latin scripts, but use local names otherwise
* always display names in the UI language

**Comment by Jean-BaptisteC** [comment]
Similar of #391

**Comment by mvglasow** [comment]
Similar but not the same, rather another use case. So far the discussion there seems to be about setting the language independently from the OS, and disabling translations for a few chosen languages, whereas this one is about disabling translated names altogether.

**Comment by biodranik** [comment]
How these places are rendered in other popular maps and apps?

**Comment by claudiush** [comment]
I checked Rijeka with from my German operating System and preferred browser language set to German:
- Google Maps shows only Rijeka, no traces of Italian or German alternative names anywhere in the UI even when selecting the place
- Apple Maps also shows only Rijeka

I prefer the Organic Maps UI approach actually: When I select a place I see the name in my language and native language below. This is only shown in the infobox in Organic Maps though while mvglasow seems to be asking for labelling on the map itself.

**Comment by mvglasow** [comment]
Correct, I am referring to labeling on the map itself. Search etc. is a different matter; here it may even be desirable for a place to be found by any of its names.

For Rijeka, the name Fiume is Italian, so you would need to select that as your language.

As for other maps, Google tends to translate as much as possible, whereas OpenStreetMap’s Carto style uses local names – as most OSM tile servers do, except for those aimed at a specific national audience.

For paper maps, translating seemed to be standard up to the 1980s. After that, mainstream paper maps switched to using local names, and only some specialty maps still use translated names.

As mentioned, different people may have different opinions on this, which means we need choice.

On December 8, 2022 7:22:30 PM GMT+02:00, Claudius ***@***.***> wrote:
>I checked Rijeka with from my German operating System and preferred browser language set to German:
>- Google Maps shows only Rijeka, no traces of Italian or German alternative names anywhere in the UI even when selecting the place
>- Apple Maps also shows only Rijeka
>
>I prefer the Organic Maps UI approach actually: When I select a place I see the name in my language and native language below. This is only shown in the infobox in Organic Maps though while mvglasow seems to be asking for labelling on the map itself.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/organicmaps/organicmaps/issues/4043#issuecomment-1343051901
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.

**Comment by fuerst** [comment]
As for other apps:
* Apple Maps allows switching off the use of translated names
* MapOut does not use translated names at all and there is no option to translate names
* Google Maps does not use translated names but I remember I had to switch translation off or Google itself did it

I prefere local names other translated names which makes navigation so much easier. Would love to see the options suggested by [mvglasow](https://github.com/mvglasow).
